pkg/util/channels/nonblocking.go

```diff
@@ -0,0 +1,11 @@
+package channels
+
+import "github.com/labstack/gommon/log"
```

Comment:
we don't use this logger. I think you need to pass a logger to this function because it can contain extra useful key value pairs

Reply:
I was too quick to commit and didn't look at it, shuld be fine now